.github/workflows/gradle.yml

```diff
@@ -84,9 +84,20 @@ jobs:
       with:
         fail_ci_if_error: true
 
+    - name: Cache Integration tests trends
```

Comment:
it's better to use name `history` instead of `trends`, because there was a request to track test history as well